test/onnx/test_fx_to_onnx.py

```diff
@@ -166,7 +166,10 @@ def forward(self, input):
             diagnostics.rules.fx_node_to_onnx,
             diagnostics.levels.WARNING,
             expected_error_node="aten.convolution.default",
-            expected_error_message="ValueError: Could not infer the attribute type from the elements of the passed Iterable value.",
+            # TODO(titaiwang): The expected error message comes from other packages, so
```

Comment:
Do you really need a TODO here?
Checking for exact error message from other packages that changes from version to version doesn't sound like a sustainable idea regardless of Meta-internal ONNX version.

Reply:
You are right. But only ValueError is not what we wanted to capture here. It's a reminder to refactor anyway.